### PR TITLE
RavenDB-14685 exposing events in the IDocumentStore interface

### PR DESCRIPTION
**Pull request by ppekrol**

### CODE REVIEW

File: src/Raven.Client/Documents/IDocumentStore.cs [hunk 1]
```diff
@@ -31,9 +31,13 @@ public interface IDocumentStore : IDisposalNotification
         X509Certificate2 Certificate { get; }
 
         /// <summary>
-        /// Store events
+        /// Store event
         /// </summary>
         event EventHandler<BeforeStoreEventArgs> OnBeforeStore;
+
+        /// <summary>
+        /// Session event
```
Comment (ayende) [inline src/Raven.Client/Documents/IDocumentStore.cs]: What does this comment mean?

File: src/Raven.Client/Documents/IDocumentStore.cs [hunk 2]
```diff
@@ -31,9 +31,13 @@ public interface IDocumentStore : IDisposalNotification
         X509Certificate2 Certificate { get; }
 
         /// <summary>
-        /// Store events
+        /// Store event
```
Comment (ayende) [inline src/Raven.Client/Documents/IDocumentStore.cs]: What is the usefulness of this doc comment?

Comment (ppekrol) [reply src/Raven.Client/Documents/IDocumentStore.cs]: Removed.

File: src/Raven.Client/Documents/IDocumentStore.cs (hunk 1 above)
Comment (ppekrol) [reply src/Raven.Client/Documents/IDocumentStore.cs]: Removed.

**Pull request merged**